Makefile

```diff
@@ -0,0 +1,64 @@
+PROJECT_UNIT_TESTS     =$(shell go list ./... | grep -v test | grep -v db)
+PROJECT_INT_TESTS      =$(shell go list ./db)
+PROJECT_E2E_TESTS      =$(shell go list ./test)
+LOCALSTACK_SERVICES    =dynamodb,kinesis,s3
+LOCAL_ENDPOINT_HOST    :=${LOCAL_ENDPOINT_HOST}
+LOCAL_ENDPOINT         =--endpoint-url=http://${LOCAL_ENDPOINT_HOST}
+DYNAMO_ENDPOINT        =${LOCAL_ENDPOINT}:4569
+KINESIS_ENDPOINT       =${LOCAL_ENDPOINT}:4568
+S3_ENDPOINT            =${LOCAL_ENDPOINT}:4572
+PROJ_TABLE_NAME        =resources
+PROJ_STREAM_NAME       =deposit
+PROJ_BUCKET_NAME       =taco-deposited-files
+PROJ_AWS_REGION        =us-east-1
+PROJ_AWS_ACCESS_KEY_ID =999999
+PROJ_AWS_SECRET_KEY    =999999
+PROJ_ENV_VARS          =AWS_REGION=${PROJ_AWS_REGION} AWS_ACCESS_KEY_ID=${PROJ_AWS_ACCESS_KEY_ID} AWS_SECRET_KEY=${PROJ_AWS_SECRET_KEY}
+
+default: test
+
+test: dependencies
+	$(PROJ_ENV_VARS) go test -v $(PROJECT_UNIT_TESTS)
+
+compose_tests:
+	$(PROJ_ENV_VARS) go test -v $(PROJECT_INT_TESTS)
+	$(PROJ_ENV_VARS) go test -v $(PROJECT_E2E_TESTS)
+
+dependencies:
+	go get github.com/golang/dep/cmd/dep
```

Comment:
I'm not thrilled about needing to run `go get` every time I want to run tests. Is there another way?